docs/UserGuide.adoc

```diff
@@ -117,7 +117,18 @@ Examples:
 Adds a meme with the image from `/Users/Me/Downloads/dgirl_oof.jpg` and tag `Funny`
 
 ****
-Note the double `/` near the prefix `p/`. The first `/` is part of the argument prefix, whereas the second `/` is part of the file path. Both `/` s must be present for the command to succeed.
+* The `PATH` argument can be either a relative path or an absolute path.
+** On MacOS and *nix systems
+*** Absolute paths start with `/` (e.g. `/home/me/Downloads/doge.jpg`).
+*** Relative paths start directly with the file/directory name (e.g. `data/images/doge.jpg`), and is resolved in the current working directory.
+** On Windows
+*** Absolute paths start with a drive letter (e.g. `C:/Users/me/Downloads`)
+*** Relative paths are the same as those on MacOS and *nix systems.
+* Note the double `/` near the prefix `p/` in the example given.
+This is because the examples in this guide use MacOS absolute file paths, which start with a `/`.
+Therefore, the first `/` is part of the argument prefix, whereas the second `/` is part of the file path.
+If you are using MacOS or *nix systems, both `/` s must be present if your path is an absolute path.
```

Comment:
```suggestion
If you are using MacOS or *nix systems, both `/` s must be present if your path is an absolute path. (e.g. p//Users/me/Downloads)
```

Reply:
:rofl: This is explaining the example given 10 lines above

Reply:
what i meant was to show them that "hey! you actually need to type p// for file path! It looks dumb but ya just do it!"

Reply:
Do you notice that this is under the same bullet point as line 127? This is doing exactly what you suggest - explaining why they must have both slashes.

Reply:
oops ok... LGTM